modules/disabling-insights-operator-gather.adoc

```diff
@@ -28,52 +34,54 @@ endif::openshift-rosa,openshift-dedicated[]
 
 .Procedure
 
-. Navigate to *Administration* -> *CustomResourceDefinitions*.
-. On the *CustomResourceDefinitions* page, use the *Search by name* field to find the *InsightsDataGather* resource definition and click it.
+. Navigate to *Administration* > *CustomResourceDefinitions*.
+. On the *CustomResourceDefinitions* page, use the *Search by name* field to find the *InsightsDataGather* custom resource definition (CRD), and click to open.
 . On the *CustomResourceDefinition details* page, click the *Instances* tab.
 . Click *cluster*, and then click the *YAML* tab.
-. Disable the gather operations by performing one of the following edits to the `InsightsDataGather` configuration file:
-.. To disable all the gather operations, enter `all` under the `disabledGatherers` key:
+. Edit the `InsightsDataGather` CRD, and complete one of the following steps:
+** To disable all the gather operations and data collection, define the `gatherers` specification and set the `mode` to *None* as outlined in the following example extract:
 +
 [source,yaml]
 ----
-apiVersion: config.openshift.io/v1alpha1
+
+apiVersion: insights.openshift.io/v1alpha2
 kind: InsightsDataGather
 metadata:
-....
-
-spec: <1>
-  gatherConfig:
-    disabledGatherers:
-      - all <2>
+  name: cluster
+spec:
+# Gatherers configuration
+  gatherers:
+    mode: None # Options: All, None, Custom
 ----
-+
---
-<1> The `spec` parameter specifies gather configurations.
-<2> The `all` value disables all gather operations.
---
-
-.. To disable individual gather operations, enter their values under the `disabledGatherers` key:
+** To disable individual gather operations, under `gatherers`, set the `mode` to *Custom* and then specify the individual gatherer that you intend to disable. For example, to disable the workload gatherer, define the following specification:
 +
 [source,yaml]
 ----
+apiVersion: insights.openshift.io/v1alpha2
+kind: InsightsDataGather
+metadata:
+  name: cluster
 spec:
-  gatherConfig:
-    disabledGatherers:
-      - clusterconfig/container_images <1>
-      - clusterconfig/host_subnets
-      - workloads/workload_info
+    # Gatherers configuration
+  gatherers:
+    mode: Custom  # Options: All, None, Custom
+    custom:
+      configs:
+        # Essential cluster configuration gatherers
+        - name: clusterconfig/authentication
+          state: Enabled
+        - name: clusterconfig/clusteroperators
+          state: Enabled
+        - name: workloads
+          state: Disabled
 ----
-+
---
-<1> Example individual gather operation
---
-+
 . Click *Save*.
-+
-After you save the changes, the Insights Operator gather configurations are updated and the operations will no longer occur.
+
+.Results
```

Comment:
I'm unsure if this heading is supported with the migration. Maybe consider tagging the sentence with the "Save" step?

Reply:
In this doc, it's confirmed a blick header of `.Results` is supported: https://docs.google.com/document/d/13NAUVAby1y1qfT77QFIZrMBhi872e7IEvAC9MUpGXbQ/edit?tab=t.0